serve.py

```diff
@@ -0,0 +1,54 @@
+import grpc
+from concurrent import futures
+import time
+import os
+
+#import the generated class
+from proto import user_pb2_grpc
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+# import logger
+from log.logging_func import logger
+
+from controller.user_controller import UserController
+
+from server import UserServicer
+
+host = os.environ.get('POSTGRES_HOST')
+database = os.environ.get('POSTGRES_DB_NAME')
+user = os.environ.get('POSTGRES_USER')
+password = os.environ.get('POSTGRES_PASSWORD')
+
+db_link = 'postgresql://' + user + ':' + password + '@' + host + '/' + database
+
+engine = create_engine(db_link)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+
+user_controller = UserController(session)
```

Comment:
this should also be taking in a logger, This way it inherits the settings of the logger defined in the program. Also, in a situation where the logger changes, it will only be changed in one place as `UserController` will only care that it received a variable that satisfies some methods that allow it to log. Doesn't care what type of logger it is etc...

Reply:
on second look, `UserController` should not be logging as the logs should be on the highest level, which is the server(server.py) level in this application

Reply:
Okay. cool for me.